packages/turbo-codemod/src/transforms/add-default-outputs.ts

```diff
@@ -0,0 +1,53 @@
+import { Flags } from "../types";
+import path from "path";
+import fs from "fs-extra";
+import { error, ok, skip } from "../logger";
+
+const DEFAULT_OUTPUTS = ["dist/**", "build/**"];
```

Comment:
This is why I mentioned the `dist/**/*` thing in the other PR.

Reply:
updated, so we don't have to validate the difference right now

Reply:
updated, so we don't have to validate the difference right now